File: version/src/main/java/com/gitee/jack20081117/frog/brain/Cell.java
package com.gitee.jack20081117.frog.brain;

/**
 * Cell is a brain nerve cell,which is the basic unit of frog's brain
 */

public class Cell {
    public int group;//this cell belongs to which group?
    public Input[] inputs;//inputs of cell,float array format like {(x1,y1),(x2,y2),...}
    public Output[] outputs;//outputs of cell,float array format like {(x1,y1),(x2,y2),...}

    public void activate(){
        for(Input xy:inputs){
        }
    }
}

File: history/version1/brain/Cell.py
# Cell.py
# Cell是神经元,属性有:神经元属于的小组编号,神经元输入区,神经元输出区

class Cell(object):
    def __init__(self):
        self.group=0
        self.inputs=[] #神经元输入区
        self.outputs=[] #神经元输出区